src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,17 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.PasswordValidation;
+
+@PasswordValidation
 public class UserRequestDto {
+    @Email
     private String email;
+    @Size(min = 6, message = "Password must be longer then 6")
     private String password;
+    @Size(min = 6, message = "Password must be longer then 6")
+    private String repeatPassword;
```

Comment:
Move all check to one place - @PasswordValidation
```suggestion
    private String password;
    private String repeatPassword;
```
![image](https://user-images.githubusercontent.com/20326858/233958347-1a913f4f-5674-426e-8fec-f2e16d29a819.png)